nature/commands/cat.lua

```diff
@@ -9,6 +9,8 @@ commander.register('cat', function(args, sinks)
 usage: cat [file]...]]
 	end
 
+	local chunk_size = 2^13 -- 8K buffer size
```

Comment:
camelCase :]

Reply:
Yeah, I missed that part of the CONTRIBUTING.md. Fair enough :)